tools/azure-sdk-tools/ci_tools/functions.py

```diff
@@ -5,6 +5,7 @@
 import tarfile
 import stat
 from ast import Not
+import coverage.exceptions
 from packaging.specifiers import SpecifierSet
```

Comment:
Redundant with import inside of `get_total_coverage`
```suggestion
from packaging.specifiers import SpecifierSet
```